python_bindings/bindings.cpp

```diff
@@ -489,7 +477,7 @@ class Index {
       assert_true(appr_alg->ef_construction_ == d["ef_construction"].cast<size_t>(), "Invalid value of ef_construction_ ");
 
       appr_alg->ef_ = d["ef"].cast<size_t>();
-      appr_alg->has_deletions_=d["has_deletions"].cast<bool>();
+      appr_alg->num_deleted_=d["num_deleted"].cast<size_t>();
```

Comment:
I wonder, can this cause errors if an old index is depickled?
Maybe add a pickle version or, at least, add a note in readme.

Reply:
I added a code to load index from old pickle